src/frontend/src/common/api/index.tsx

```diff
@@ -26,8 +27,12 @@ export enum ORG_API {
 }
 
 export const generateOrgSpecificUrl = (path: ORG_API): string => {
```

Comment:
should we rename to `generateOrgPathogenSpecificUrl`? or may `generateContextSpecificUrl`?

Reply:
also should maybe consider renaming `ORG_API` to `ORG_PATHOGEN_API`, but that may be a little out of scope for this